src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -150,8 +193,9 @@ export default {
 		}
 		&__right {
 			justify-self: flex-start;
+			justify-content:  space-between;
 			position: relative;
-			flex: 0 0 110px;
+			flex: 0 0 120px;
```

Comment:
Careful with specific values like that, Ideally they should be indexed on one of your existing variable :)